src/CMakeLists.txt

```diff
@@ -45,9 +45,27 @@ endif()
 add_library(cucumber-cpp-nomain STATIC ${CUKE_SOURCES})
 add_library(cucumber-cpp STATIC ${CUKE_SOURCES} main.cpp)
 
+target_include_directories(cucumber-cpp-nomain PUBLIC
+    $<BUILD_INTERFACE:${CUKE_INCLUDE_DIR}>
```

Comment:
Is adding this enough to permit us from removing the directory-scoped `include_directories(${CUKE_INCLUDE_DIR})` in the toplevel `CMakeLists.txt`?